client/src/containers/ConsumerGroup/ConsumerGroupDetail/ConsumerGroupUpdate/ConsumerGroupUpdate.jsx

```diff
@@ -133,60 +148,71 @@ class ConsumerGroupUpdate extends Form {
     this.setState({ formData });
   };
 
-  createSubmitBody = formData => {
+  createSubmitBody = (formData, checked) => {
     let body = [];
     let splitName = [];
     let topic = '';
     let partition = '';
 
-    console.log(formData)
-
     Object.keys(formData).forEach(name => {
-      splitName = name.split('-');
-      partition = splitName.pop();
-      topic = splitName.join('-');
-
-      body.push({
-        topic,
-        partition,
-        offset: formData[name]
-      });
+        splitName = name.split('-');
+        partition = splitName.pop();
+        topic = splitName.join('-');
+
+      if (checked[topic] === true) {
+        body.push({
+          topic,
+          partition,
+          offset: formData[name]
+        });
+      }
     });
 
-    console.log(body)
-
     return body;
   };
 
   async doSubmit() {
-    const { clusterId, consumerGroupId, formData } = this.state;
+    const { clusterId, consumerGroupId, formData, checked } = this.state;
 
     await post(
       uriConsumerGroupUpdate(clusterId, consumerGroupId),
-      this.createSubmitBody(formData)
+      this.createSubmitBody(formData, checked)
     );
 
     this.setState({ state: this.state });
     toast.success(`Offsets for '${consumerGroupId}' updated successfully.`);
   }
 
   renderGroupedTopicOffset = () => {
-    const { groupedTopicOffset } = this.state;
+    const { groupedTopicOffset, checked } = this.state;
     const renderedItems = [];
 
-    Object.keys(groupedTopicOffset).forEach(topidId => {
+    Object.keys(groupedTopicOffset).forEach(topicId => {
       renderedItems.push(
-        <fieldset id={`fieldset-${topidId}`} key={topidId}>
-          <legend id={`legend-${topidId}`}>{topidId}</legend>
-          {this.renderPartitionInputs(groupedTopicOffset[topidId], topidId)}
+        <fieldset id={`fieldset-${topicId}`} key={topicId}>
+          <legend id={`legend-${topicId}`}>
+            <input
+              type="checkbox"
+              value={topicId}
+              defaultChecked={checked[topicId]}
+              onChange={this.checkedGroupedTopicOffset}/> {topicId}
```

Comment:
@polarising-java : need some help here please :pray: 
I need to click several time on the checkbox to make it work ! 
don't understand, the uncheck all button works fine !